CONTRIBUTING.md

```diff
@@ -65,10 +65,58 @@ the [puppet-dev mailing list](https://groups.google.com/forum/#!forum/puppet-dev
       why this is a problem, and how the patch fixes the problem when applied.
   ```
 * Make sure you have added the necessary tests for your changes.
-* Run _all_ the tests to assure nothing else was accidentally broken. First
-  install all the test dependencies with `bundle install --path .bundle`. Then
-  either run all the tests serially with `bundle exec rspec spec` or in parallel
-  with `bundle exec rake parallel:spec[process_count]`
+
+## Running Tests
+Before you submit your pull request, please ensure all unit tests pass. This
+includes any tests you may have added or modified in addition to all existing
+unmodified tests.
+
+The general workflow is as follows:
+* Ensure ruby and bundler are installed
+* Install dependencies specified with bundler
+  `bundle install --path .bundle`
+* Run the tests. You have a few different options on how to run them:
+  * Run one test by specifying the path to the test
+    `bundle exec rspec spec/unit/puppet_spec.rb`
+  * Run all tests serially
+    `bundle exec rspec`
+  * Run all tests in parallel. Here `4` is the number of processors to use when
+  running tests. This number will be different based on where you are running
+  these tests.
+    `bundle exec rake parallel:spec[4]`
+
+#### On Windows
```

Comment:
Some of this info is repeated in https://github.com/puppetlabs/puppet/blob/master/docs/windows.md. Could you DRY it?

Reply:
Also https://github.com/puppetlabs/puppet/blob/master/docs/rspec_tutorial.md

Reply:
What do you mean by `DRY`?

Reply:
ah yeah, DRY = "don't repeat yourself" as we describe how to run tests in `docs/quickstart.md`, `docs/windows.md` and `CONTRIBUTING.md`